src/html.js

```diff
@@ -11,9 +12,37 @@ export default function HTML(props) {
                     name="viewport"
                     content="width=device-width, initial-scale=1, shrink-to-fit=no"
                 />
+                <script
+                    dangerouslySetInnerHTML={{
+                        __html: `
+                    if (window.location.hostname === 'deriv-com.binary.sx') gtm_id = 'GTM-TNX2ZKH'
+                    (function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':
+                    new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0],
+                    j=d.createElement(s),dl=l!='dataLayer'?'&l='+l:'';j.async=true;j.src=
+                    'https://www.googletagmanager.com/gtm.js?id='+i+dl;f.parentNode.insertBefore(j,f);
+                    })(window,document,'script','dataLayer', gtm_id)`,
+                    }}
+                />
+                <script async src={'https://www.googletagmanager.com/gtm.js?id=' + gtm_id}></script>
```

Comment:
i m not sure whether it will get the hoisting correctly, but shouldnt this be above the create element script